remix-gql-tailwind/.env.example

```diff
@@ -1,2 +1,6 @@
 NODE_ENV="development"
 SESSION_SECRET="super-duper-s3cret"
+GITHUB_CLIENT_ID=
+GITHUB_CLIENT_SECRET=
```

Comment:
A couple notes:
1. If we put these in GitHub secrets for whatever reason, they need different names as these are reserved in GitHub actions.
2. For a valid secrets file, we should put at least `xxx` on the right hand of the equals